sdks/python/apache_beam/runners/interactive/background_caching_job.py

```diff
@@ -19,21 +19,21 @@
 
 For internal use only; no backwards-compatibility guarantees.
 
-A background caching job is a job that captures events for all capturable
+A background caching job is a job that records events for all recordable
```

Comment:
"Background caching job" -> "Background source recording job". Please check all occurrences.

Reply:
Switched to background source recording jobs for all the comments.